packages/liveblocks-client/src/json.ts

```diff
@@ -12,3 +12,28 @@ export type Json = JsonScalar | JsonArray | JsonObject;
 export type JsonScalar = string | number | boolean | null;
 export type JsonArray = Json[];
 export type JsonObject = { [key: string]: Json };
+
+/**
+ * Alternative to JSON.parse() that will not throw in production. If the passed
+ * string cannot be parsed, this will return `undefined`.
+ */
+export function parseJson(rawMessage: string): Json | undefined {
+  try {
+    // eslint-disable-next-line no-restricted-syntax
+    return JSON.parse(rawMessage);
+  } catch (e) {
+    return undefined;
+  }
+}
+
+export function isJsonArray(data: Json): data is JsonArray {
+  return Array.isArray(data);
+}
+
+export function isJsonObject(data: Json): data is JsonObject {
+  return data !== null && typeof data === "object" && !isJsonArray(data);
+}
+
+export function isJsonScalar(data: Json): data is JsonScalar {
```

Comment:
I don't think it's used right now

Reply:
You're right! Removed in a8e7a14fa7cf1a26275161780af020efb2ce4c6c 👍